app/views/org_admin/templates/index.html.erb

```diff
@@ -66,19 +69,21 @@
         <% end %>
       </div>
       <!-- If the Org is not just a funder then show the customizations table -->
-      <div id="funder-templates" role="tabpanel" class="tab-pane">
-        <h2><%= _('Customizable Templates') %></h2>
-        <% if customizable_templates.length > 0 %>
-          <%= paginable_renderise(
-            partial: 'funder_templates_list',
-            controller: 'org_admin/templates',
-            action: 'funders', 
-            scope: customizable_templates,
-            locals: {current_org: current_org.id, customizations: customized_templates, published: published, scopes: scopes[:funders]}) %>
-        <% else %>
-          <p><%= _('There are currently no customisable templates.') %></p>
-        <% end %>
-      </div>
+      <% if !current_org.funder_only? %>
+        <div id="funder-templates" role="tabpanel" class="tab-pane">
+          <h2><%= _('Customizable Templates') %></h2>
+          <% if customizable_templates.length > 0 %>
+            <%= paginable_renderise(
+              partial: 'funder_templates_list',
+              controller: 'org_admin/templates',
+              action: 'funders', 
+              scope: customizable_templates,
+              locals: {current_org: current_org.id, customizations: customized_templates, published: published, scopes: scopes[:funders]}) %>
+          <% else %>
+            <p><%= _('There are currently no customisable templates.') %></p>
```

Comment:
With my latest PR for paginable, the partial funder_templates_list will not be rendered if scope.length is zero, therefore you can remove the guard that checks for length as well as the info message.

Reply:
ok will do

Reply:
ok will remove